packages/dmn-js-decision-table/src/features/view-drd/ViewDrd.js

```diff
@@ -1,12 +1,13 @@
 import ViewDrdComponent from './components/ViewDrdComponent';
 
 export default class ViewDrd {
-  constructor(components, eventBus, injector, sheet) {
+  constructor(components, eventBus, injector, sheet, config) {

```

Comment:
this will always be `undefined` because you did not declare `config` in the injected modules, cf. https://github.com/bpmn-io/dmn-js/blob/b65fe68c1cc09822904dde5864248835199697ae/packages/dmn-js-decision-table/src/features/view-drd/ViewDrd.js#L48

Reply:
After adding 'config', there's an error from the test written before: "view drd should show view drd button FAILED" AssertionError: expected null to exist, so I think the domQuery for the view-drd-button returns null. I don't know if I missed anything from the injected module, config part or the test? And do the MockViewer vs DmnDecisionTableViewer have anything to do with this problem?

Reply:
Can you reference the test case you mean?